proj/hps_accel/gateware/hps_cfu.py

```diff
@@ -36,10 +40,40 @@ def elab(self, m):
             m.d.sync += self.done.eq(0)
 
 
+class AddOneActor(BinaryCombinatorialActor):
+    """A binary actor that adds one to a stream of integers.
+
+    Used to implement the verification register.
+    """
+
+    def __init__(self):
+        super().__init__(unsigned(32), unsigned(32))
+
+    def transform(self, m, input, output):
+        m.d.comb += output.eq(input + 1)
```

Comment:
It obviously doesn't matter greatly, but the comment for REG_VERIFY in constants.py says that it will negate its input value, not add 1.